integration-tests/test-harness/src/screens/MiscTestingScreen.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import {Button, View, Text} from "react-native";
+import {useCallback} from "react";
+import {styles} from "../helpers/styles";
+import {
+  getDeviceId,
+  getLastRunEndState,
+  getCurrentSessionId,
+  recordNetworkRequest,
+  logNetworkClientError,
+} from "@embrace-io/react-native";
+
+const MiscTestingScreen = () => {
+  const getInfo = useCallback(async () => {
+    try {
+      const deviceId = await getDeviceId();
+      const sessionId = await getCurrentSessionId();
+      const lastRunEndState = await getLastRunEndState();
+      console.log("deviceId: ", deviceId);
+      console.log("sessionId: ", sessionId);
+      console.log("lastRunEndState: ", lastRunEndState);
+    } catch (e) {
+      console.log("failed to get info from SDK");
+    }
+  }, []);
+
+  const networkRequestsHandler = useCallback(async () => {
```

Comment:
fyi these handlers duplicate some code from the [NSF pr](https://github.com/embrace-io/embrace-react-native-sdk/pull/268/files#diff-d6a88963bd720b3fd7b3e93e23a0394c5d6539802959629167f4f1c601d41106), but I can take a look before merging and clean some stuff

Reply:
ok sounds good